lib/MusicBrainz/Server/Controller/Root.pm

```diff
@@ -347,6 +366,7 @@ sub begin : Private
         new_edit_notes => $new_edit_notes,
         new_edit_notes_mtime => $new_edit_notes_mtime,
         contact_url => $CONTACT_URL,
+        main_server_url => $main_server_url->as_string,
```

Comment:
this should probably be `defined $main_server_url ? $main_server_url->as_string : undef,` since it's only set for staging servers